cypress/e2e/api/engageSphere.cy.js

```diff
@@ -0,0 +1,135 @@
+      expect(response.status).to.eq(200);
+      expect(response.body).to.have.property('pageInfo');
+      expect(response.body.pageInfo).to.deep.include({
+        currentPage: 2,
+      });
+    });
+  });
+
+  it('Filtra corretamente o limite de clientes', () => {
+    const limit = 2;
+
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: {
+        page: 2,
+        limit: limit,
+        size: 'All',
+        industry: 'All'
+      }
+    }).then((response) => {
+      const totalCustomers = response.body.pageInfo.totalCustomers;
+      const expectedTotalPages = Math.ceil(totalCustomers / limit);
+
+      expect(response.body.pageInfo).to.deep.include({
+        totalPages: expectedTotalPages,
+        totalCustomers: totalCustomers
+      });
+    });
+  });
+
+  it('Trata requisições inválidas (por exemplo, página negativa)', () => {
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: { page: -1, limit: 5, size: 'All', industry: 'All' },
+      failOnStatusCode: false
+    }).then((response) => {
+      expect(response.status).to.eq(400);
+      expect(response.body).to.have.property('error', 'Invalid page or limit. Both must be positive numbers.');
+    });
+  });
+
+  it('Trata requisições inválidas (por exemplo, limite negativo)', () => {
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: { page: 1, limit: -5, size: 'All', industry: 'All' },
+      failOnStatusCode: false
+    }).then((response) => {
+      expect(response.status).to.eq(400);
+      expect(response.body).to.have.property('error', 'Invalid page or limit. Both must be positive numbers.');
+    });
+  });
+
+  it('Trata requisições inválidas (por exemplo, página como string)', () => {
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: { page: 'abc', limit: 5, size: 'All', industry: 'All' },
+      failOnStatusCode: false
+    }).then((response) => {
+      expect(response.status).to.eq(400);
+      expect(response.body).to.have.property('error', 'Invalid page or limit. Both must be positive numbers.');
+    });
+  });
+
+  it('Trata requisições inválidas (por exemplo, limite como booleano)', () => {
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: { page: 1, limit: false, size: 'All', industry: 'All' },
+      failOnStatusCode: false
+    }).then((response) => {
+      expect(response.status).to.eq(400);
+      expect(response.body).to.have.property('error', 'Invalid page or limit. Both must be positive numbers.');
+    });
+  });
+
+  it('Trata requisições inválidas (por exemplo, tamanho não suportado)', () => {
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: {
+        page: 1,
+        limit: 5,
+        size: 'InvalidSize', // Valor não suportado
```

Comment:
Precisa mesmo desse comentário? Seu valor já deixa isso claro, não?

```suggestion
        size: 'InvalidSize',
```